### PR TITLE
docs: improve README examples of `stats/base/dists/normal` namespace

### LINKED ISSUE #1638: [RFC]: improve README examples of `stats/base/dists/normal` namespace
### Description

This RFC proposes to improve the [README examples](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/dists/normal#examples) of the `stats/base/dists/normal` namespace package. Currently, namespace package READMEs only include minimal examples which are not particularly informative and do not showcase namespace functionality.

Historically, namespace examples were not given much effort due to the rapidly changing nature of the project; however, now that development and organization has settled, it would be good to revisit these packages and add proper examples to better communicate to users how to use namespace contents.

For those wanting to contribute to this issue, you can use the following steps:

- Browse through the [contents](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/dists/normal) of the `stats/base/dists/normal` namespace and identify interesting functions and packages to showcase.
- Update the example code block in the "Examples" section of the README for the `stats/base/dists/normal` namespace and open a PR.
- When updating the README, examples should follow existing project conventions (e.g., ES5).
- Examples may `require` functionality from other packages to help make examples more interesting.

### Related Issues

None.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by nightknighto**
Resolves #1638

## Description

> What is the purpose of this pull request?

This pull request:

- Add examples of the functions present in `stats/base/dists/normal` namespace and their usage.
- Adds 4 example sets in the README.
- Adds 4 example js files corresponding to the new sets.

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #1638
-   fixes #1638

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

I've made a separate js file for each example set to not put all codes together and to make it more tidy, manageable and independent.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/stats/base/dists/normal/README.md [hunk 1]
```diff
@@ -115,6 +115,96 @@ var normal = require( '@stdlib/stats/base/dists/normal' );
 console.log( objectKeys( normal ) );
 ```
 
+```javascript
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/normal/README.md]: Prefer instead to modify the existing example. Project conventions are not to separate out examples into several consecutive Markdown code blocks.

Comment (nightknighto) [reply lib/node_modules/@stdlib/stats/base/dists/normal/README.md]: Merged all blocks, and did the same for example js files

Comment (nightknighto) [reply lib/node_modules/@stdlib/stats/base/dists/normal/README.md]: @kgryte What else do I need for this PR?

**Comment by nightknighto** [comment]
@Planeshifter @kgryte

**Review by Planeshifter — APPROVED** [review]
Thanks @Deadreyo!

I have added an overarching scenario for the example code and updated the code style to follow project conventions (e.g., use spaces around function parentheses.)

Will merge once CI checks have passed.

**Pull request merged**